liiatools_pipeline/ops/common_la.py

```diff
@@ -0,0 +1,252 @@
+from dagster import Config
+import re
+import pandas as pd
+from typing import List, Tuple
+from dagster import In, Out, op
+from fs.base import FS
+
+from liiatools.common import pipeline as pl
+from liiatools.common.archive import DataframeArchive
+from liiatools.common.checks import check_year_within_range
+from liiatools.common.constants import SessionNames, SessionNamesFixEpisodes
+from liiatools.common.data import FileLocator, ErrorContainer, DataContainer
+from liiatools.common.reference import authorities
+from liiatools.common.stream_errors import StreamError
+from liiatools.common.transform import degrade_data, enrich_data, prepare_export
+from liiatools.cin_census_pipeline.spec import load_schema as load_schema_cin
+from liiatools.cin_census_pipeline.stream_pipeline import task_cleanfile as task_cleanfile_cin
+from liiatools.ssda903_pipeline.spec import load_schema as load_schema_ssda903
+from liiatools.ssda903_pipeline.stream_pipeline import task_cleanfile as task_cleanfile_ssda903
+from liiatools.ssda903_pipeline.fix_episodes import stage_1, stage_2
+
+
+from liiatools_pipeline.assets.common import (
+    incoming_folder,
+    workspace_folder,
+    shared_folder,
+    pipeline_config,
+    la_code as input_la_code,
+    dataset,
+)
+
+
+class FileConfig(Config):
+    filename: str
+    name: str
+
+
+@op(
+    out={
+        "session_folder": Out(FS),
+        "session_id": Out(str),
+        "incoming_files": Out(List[FileLocator]),
+    }
+)
+def create_session_folder() -> Tuple[FS, str, List[FileLocator]]:
+    session_folder, session_id = pl.create_session_folder(
+        workspace_folder(), SessionNames
+    )
+    incoming_files = pl.move_files_for_processing(incoming_folder(), session_folder)
+
+    return session_folder, session_id, incoming_files
+
+
+@op(
+    out={"current": Out(DataframeArchive)},
+)
+def open_current() -> DataframeArchive:
+    current_folder = workspace_folder().makedirs("current", recreate=True)
+    current = DataframeArchive(current_folder, pipeline_config(), "dataset()")
```

Comment:
need to remove "" from dataset() otherwise it will just be the string dataset() rather than the actual value